src/main/java/org/folio/rest/utils/HelperUtils.java

```diff
@@ -21,5 +24,31 @@ public static void respond(Handler<AsyncResult<Response>> handler, Response resp
     handler.handle(result);
   }
 
-
+  public enum SequenceQuery {
+
+    CREATE_SEQUENCE {
+      @Override
+      public String getQuery(String purchaseOrderId) {
+        return "CREATE SEQUENCE " + constructSequenceName(purchaseOrderId) + " MINVALUE 1 MAXVALUE 999";
```

Comment:
See the comments in https://wiki.folio.org/display/FOLIJET/Postgres+SEQUENCEs+for+generating+po_line_number+suffixes

I think we probably want to be safe and add an IF NOT EXISTS clause.

Reply:
@craigmcnally,